spylon_kernel/scala_kernel.py

```diff
@@ -85,14 +142,47 @@ def do_execute_direct(self, code, silent=False):
             return self.Error(e.scala_message)
 
     def get_completions(self, info):
+        """Gets completions from the kernel based on the provided info.
+
+        Part of the MetaKernel API.
+
+        Parameters
+        ----------
+        info : dict
+            Information returned by `metakernel.parser.Parser.parse_code`
+            including `code`, `help_pos`, `start`, etc.
+
+        Returns
+        -------
+        list of str
+            Possible completions for the code
+        """
         return self._scalamagic.get_completions(info)
 
     def get_kernel_help_on(self, info, level=0, none_on_fail=False):
+        """Gets help text for the `info['help_obj']` identifier.
+
+        Part of the MetaKernel API.
+
+        Parameters
+        ----------
+        info : dict
+            Information returned by `metakernel.parser.Parser.parse_code`
+            including `help_obj`, etc.
+        level : int
+            Level of help to request, 0 for basic, 1 for more, etc.
+        none_on_fail : bool
+            Ignored
```

Comment:
Ignored down in `_scalamagic.get_help_on`. I'll try to figure out from the metakernel source exactly how it's used.